lib/class-wp-rest-plugins-controller.php

```diff
@@ -61,11 +72,63 @@ public function delete_item( $request ) {
 	}
 
 	public function prepare_item_for_response( $item, $request ) {
-
+		return $item;
 	}
 
 	public function get_item_schema() {
-
+		$schema = array(
+			'$schema'              => 'http://json-schema.org/draft-04/schema#',
+			'title'                => 'plugin',
+			'type'                 => 'object',
+			'properties'           => array(
+				'Name'        => array(
```

Comment:
Is `title` an unique identifier of the item? If it is, then it'll be the slug I think (otherwise, I can remove `Name`).